### ISSUE TITLE
i trained on my own dataset,bu the accuracy is 0.058

### ISSUE THREAD

**Issue by yulijun1234** [kevinlin311tw/Caffe-DeepBinaryCode#29]
I1116 18:09:20.180027  9262 solver.cpp:258]     Train net output #2: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:09:20.180032  9262 solver.cpp:571] Iteration 49500, lr = 0.0001
I1116 18:09:33.337271  9262 solver.cpp:242] Iteration 49600, loss = 2.20369
I1116 18:09:33.337370  9262 solver.cpp:258]     Train net output #0: loss: 50%-fire-rate = 0.078342 (* 1 = 0.078342 loss)
I1116 18:09:33.337389  9262 solver.cpp:258]     Train net output #1: loss: classfication-error = 2.25035 (* 1 = 2.25035 loss)
I1116 18:09:33.337394  9262 solver.cpp:258]     Train net output #2: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:09:33.337399  9262 solver.cpp:571] Iteration 49600, lr = 0.0001
I1116 18:09:46.505617  9262 solver.cpp:242] Iteration 49700, loss = 4.80791
I1116 18:09:46.505666  9262 solver.cpp:258]     Train net output #0: loss: 50%-fire-rate = 0.078342 (* 1 = 0.078342 loss)
I1116 18:09:46.505671  9262 solver.cpp:258]     Train net output #1: loss: classfication-error = 4.85457 (* 1 = 4.85457 loss)
I1116 18:09:46.505676  9262 solver.cpp:258]     Train net output #2: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:09:46.505679  9262 solver.cpp:571] Iteration 49700, lr = 0.0001
I1116 18:09:59.674759  9262 solver.cpp:242] Iteration 49800, loss = 4.90798
I1116 18:09:59.674805  9262 solver.cpp:258]     Train net output #0: loss: 50%-fire-rate = 0.078342 (* 1 = 0.078342 loss)
I1116 18:09:59.674811  9262 solver.cpp:258]     Train net output #1: loss: classfication-error = 4.95464 (* 1 = 4.95464 loss)
I1116 18:09:59.674815  9262 solver.cpp:258]     Train net output #2: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:09:59.674820  9262 solver.cpp:571] Iteration 49800, lr = 0.0001
I1116 18:10:12.822674  9262 solver.cpp:242] Iteration 49900, loss = 4.80472
I1116 18:10:12.822844  9262 solver.cpp:258]     Train net output #0: loss: 50%-fire-rate = 0.078342 (* 1 = 0.078342 loss)
I1116 18:10:12.822870  9262 solver.cpp:258]     Train net output #1: loss: classfication-error = 4.85138 (* 1 = 4.85138 loss)
I1116 18:10:12.822875  9262 solver.cpp:258]     Train net output #2: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:10:12.822898  9262 solver.cpp:571] Iteration 49900, lr = 0.0001
I1116 18:10:25.875818  9262 solver.cpp:449] Snapshotting to binary proto file SSDH48_iter_50000.caffemodel
I1116 18:10:26.749424  9262 solver.cpp:734] Snapshotting solver state to binary proto fileSSDH48_iter_50000.solverstate
I1116 18:10:27.045744  9262 solver.cpp:326] Iteration 50000, loss = 2.2021
I1116 18:10:27.045768  9262 solver.cpp:346] Iteration 50000, Testing net (#0)
I1116 18:10:39.996009  9262 solver.cpp:414]     Test net output #0: accuracy = 0.0591
I1116 18:10:39.996035  9262 solver.cpp:414]     Test net output #1: loss: 50%-fire-rate = 0.0783422 (* 1 = 0.0783422 loss)
I1116 18:10:39.996040  9262 solver.cpp:414]     Test net output #2: loss: classfication-error = 2.82341 (* 1 = 2.82341 loss)
I1116 18:10:39.996044  9262 solver.cpp:414]     Test net output #3: loss: forcing-binary = -0.125 (* 1 = -0.125 loss)
I1116 18:10:39.996060  9262 solver.cpp:331] Optimization Done.
I1116 18:10:39.996064  9262 caffe.cpp:214] Optimization Done.

my dataset is 1020 as train images and 340 as val images, i make it look like your dataset with labels, and make them into lmdb , but the result is not good , can you tell me why?thank you so much!
